libraries/platforms/aws-vcs/execution.mk

```diff
@@ -74,20 +35,6 @@ help:
 	@echo "      regression: Run all cosimulation regression tests for "
 	@echo "             this subdirectory and print pass/fail results"
 	@echo ""
-	@echo "      regression+traces: Run all cosimulation regression tests"
-	@echo "             for this subdirectory and generate traces for each"
-	@echo "             test. Does not print aggregate pass/fail results."
-	@echo ""
-	@echo "      regression+waveforms: Run all cosimulation regression "
-	@echo "             tests for this subdirectory and generate waveforms"
-	@echo "             for each test. Does not print aggregate pass/fail"
-	@echo "             results."
-	@echo ""
-	@echo "      regression+traces+waveforms: Run all cosimulation "
-	@echo "             regression tests for this subdirectory and generate"
-	@echo "             traces and waveforms for each test. Does not print"
-	@echo "             aggregate pass/fail results."
-	@echo ""
```

Comment:
I got rid of these for now. My long-term goal is to replace the regression suite with a python script (e.g. with Pytest) that a user can configure to get the same outputs.

It wasn't worth my time to update these right now.